diff --git a/node_modules/@op-engineering/op-sqlite/android/src/main/java/com/op/sqlite/OPSQLiteModule.kt b/node_modules/@op-engineering/op-sqlite/android/src/main/java/com/op/sqlite/OPSQLiteModule.kt
index fea0aa7..75a2392 100644
--- a/node_modules/@op-engineering/op-sqlite/android/src/main/java/com/op/sqlite/OPSQLiteModule.kt
+++ b/node_modules/@op-engineering/op-sqlite/android/src/main/java/com/op/sqlite/OPSQLiteModule.kt
@@ -51,7 +51,7 @@ internal class OPSQLiteModule(context: ReactApplicationContext?) : ReactContextB
     }
 
     @ReactMethod(isBlockingSynchronousMethod = true)
-    fun getDylibPath(bundleId: String, name: String) {
+    fun getDylibPath(bundleId: String, name: String): Boolean {
         throw Exception("Do not call getDylibPath on Android")
     }
 
